conda.recipe/meta.yaml

```diff
@@ -5,16 +5,23 @@ package:
 source:
   path: ../
 
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
+
```

Comment:
Based it off bqplot's as i was trying things. This seems to build fine, and
does all the "normal" file naming. That's all so that releases are concise,
while dev builds are usefully informative.

I am not sure i need all of the power, but i do like how hackable these low
level details are.

On 07:51, Mon, Nov 16, 2015 Damián Avila <notifications@github.com> wrote:

> In conda.recipe/meta.yaml
> <https://github.com/ContinuumIO/nbpresent/pull/13#discussion_r44919444>:
>
> > @@ -5,16 +5,23 @@ package:
> >  source:
> >    path: ../
> >
> > +build:
> > +  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
> > +  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
> > +  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
> > +
>
> Feels a little bit over-complicated, why do you need this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/nbpresent/pull/13/files#r44919444>.
>